src/rust/cryptography-keepalive/src/lib.rs

```diff
@@ -19,6 +20,10 @@ pub unsafe trait StableDeref: Deref {}
 // slice returned by `deref` remains valid.
 unsafe impl<T> StableDeref for Vec<T> {}
 
+// SAFETY: `PyBackedBytes`'s data is on the heap, so as long as it's not mutated, the
+// slice returned by `deref` remains valid.
```

Comment:
`bytes` are immutable in python, so the comment here should say that, not that it's fine as long as it's not mutated.

Reply:
fixed